updates/22/asset_collections_mappings_updates.sql

```diff
@@ -54,8 +54,6 @@ INSERT INTO multiasset_mappings(collection_id, asset) VALUES (248, "eip155:1/erc
 *
 INSERT INTO multiasset_mappings(collection_id, asset) VALUES (248, "eip155:100/erc20:0x1509706a6c66CA549ff0cB464de88231DDBe213B");
 *
-INSERT INTO multiasset_mappings(collection_id, asset) VALUES (23, "eip155:10/erc20:0x50c5725949A6F0c72E6C4a641F24049A917DB0Cb");
-*
```

Comment:
have you created a note to remove this one i nthe db upgrade? We will forget otherwise.

Reply:
created here https://github.com/orgs/rotki/projects/11/views/2?pane=issue&itemId=63714158